main.js

```diff
@@ -28,6 +29,14 @@ app.on("ready", () => {
       : `file://${__dirname}/build/index.html`
   );
 
+  // Get list of school types from data base.
+  ipcMain.on("db:schoolType", () => {
+    getSchoolTypeList().then(data => {
```

Comment:
we can take advantage of `async/await` syntax here to improve readability.
it could look something like that:
`
async () => {
  const data = await getSchoolTypeList();
   mainWindow.webContents.send("sendData", data)
}`

or even something like `mainWindow.webContents.send("sendData", await getSchoolTypeList())` will do the trick as well ;)